pkg/fixtures/group.go

```diff
@@ -15,7 +15,17 @@ func GetGroup() *storage.Group {
 	}
 }
 
-// GetGroups returns a set of mock storage.Group objects, which in total represents the possible combinations of group properties and roles
+// GetGroupWithMutability returns a mock storage.Group with all possible properties filled out.
+func GetGroupWithMutability(mode storage.MutabilityMode) *storage.Group {
+	group := GetGroup()
+
+	group.GetProps().Traits = &storage.Traits{MutabilityMode: mode}
```

Comment:
```suggestion
	group.Props.Traits = &storage.Traits{MutabilityMode: mode}
```
`GetProps()` only makes sense if we think that `group` could be `nil`. However, in this case, `nil` props would be returned and the `.Traits` access would cause a panic.